### PR TITLE
add documentation for the WES CLI

### PR DESCRIPTION
**Pull request by wshands**
Please make sure that you've checked the following before submitting your pull request. Thanks!

- [] Check that you pass the basic style checks and unit tests by running `mvn clean install`

### CODE REVIEW

File: _docs/advanced-features.md [hunk 1]
```diff
@@ -206,6 +206,72 @@ dockstore tool launch --local-entry Dockstore.cwl --json test.json --uuid fakeUU
...
+- --wes-url \<WES URL>                 URL where the WES request should be sent, e.g. 'http://localhost:8080/ga4gh/wes/v1'
+- --wes-auth \<auth>                   Authorization credentials for the WES endpoint, e.g. 'Bearer 12345'
+
+
+#### Config file settings
+Place WES settings after a separate '[WES]' section of the config file. At this time only 'url' and 'authorization' settings are supported.
+For example:
+```bash
+token: <my token>
+server-url: https://dockstore.org:8443
```
Comment (coverbeck) [inline _docs/advanced-features.md]: We are trying to get rid of an explicit port everywhere -- please change to `https://dockstore.org/api`

File: _docs/advanced-features.md [hunk 2]
```diff
@@ -206,6 +206,72 @@ dockstore tool launch --local-entry Dockstore.cwl --json test.json --uuid fakeUU
...
--->
\ No newline at end of file
+-->
+
+## Workflow Execution Service (WES) Command Line Interface (CLI)
+
+The Workflow Execution Service API describes a standard programmatic way to run and manage workflows, see more information here: 
+https://github.com/ga4gh/workflow-execution-service-schemas
+
+The Dockstore CLI supports a WES client that allows users to submit a request to launch a workflow run, get the status of a run, or cancel a
```
Comment (coverbeck) [inline _docs/advanced-features.md]: I would say "...implements a WES client..."

File: _docs/advanced-features.md (hunk 1 above)
Comment (wshands) [reply _docs/advanced-features.md]: done

File: _docs/advanced-features.md (hunk 2 above)
Comment (wshands) [reply _docs/advanced-features.md]: done

**Comment by wshands** [comment]
Created ticket for WES CLI tutorial at https://ucsc-cgl.atlassian.net/browse/DOCK-635

File: _docs/advanced-features.md [hunk 3]
```diff
@@ -206,6 +206,72 @@ dockstore tool launch --local-entry Dockstore.cwl --json test.json --uuid fakeUU
...
+
 <!--stackedit_data:
 eyJoaXN0b3J5IjpbMjA4MjI5MzQ4NV19
--->
\ No newline at end of file
+-->
+
+## Workflow Execution Service (WES) Command Line Interface (CLI)
+
+The Workflow Execution Service API describes a standard programmatic way to run and manage workflows, see more information here:
```
Comment (coverbeck) [inline _docs/advanced-features.md]: I think this is a run-on sentence. I suggest "...manage workflows. See more..."

File: _docs/advanced-features.md [hunk 4]
```diff
@@ -206,6 +206,72 @@ dockstore tool launch --local-entry Dockstore.cwl --json test.json --uuid fakeUU
...
+```
+
+- Cancel a run (a run id is returned in the response from launching a WES workflow run):
+```bash
+dockstore workflow wes cancel --id <run id>
+```
+
+
+### WES Endpoint and Authorization
+Dockstore WES CLI requests will be sent to the WES endpoint specified in the Dockstore config file and will use authorization credentials
```
Comment (coverbeck) [inline _docs/advanced-features.md]: Suggest  "By default Dockstore WES CLI requests will be sent..."

**Pull request merged**